client/.env

```diff
@@ -1 +1,11 @@
-EXTEND_ESLINT=true
```

Comment:
@Serhazor Please be very careful in the future - it looks like you have accidentally committed secrets to git. I am going to go ahead and rebase to remove these secrets from the git history. Also, these environment variables are for the backend `.env`, not the frontend.